example_spec.rb

```diff
@@ -16,3 +16,8 @@
 caller.first
 caller_locations[1]
 caller_locations.first
+##Patterns: Layout_ArrayAlignment
+
+menu_item = ['Spam', 'Spam',
```

Comment:
Codacy found an issue: [Useless assignment to variable - `menu_item`.](https://app.staging.codacy.org/manual/josemiguelmelo/example-rubocop/pullRequest?prid=9434)